src/java/org/apache/cassandra/streaming/StreamDeserializingTask.java

```diff
@@ -109,6 +111,12 @@ public StreamSession deriveSession(StreamMessage message)
         // Attach this channel to the session: this only happens upon receiving the first init message as a follower;
         // in all other cases, no new control channel will be added, as the proper control channel will be already attached.
         streamSession.attachInbound(channel);
+
+        if (streamSession.getStreamOperation() == StreamOperation.BULK_LOAD && !Guardrails.bulkLoadEnabled.isEnabled(null))
```

Comment:
We tend to omit braces when bodies of `if`s contain just one statement. In this case you do not need braces here.